### ISSUE TITLE
Update dktest

### ISSUE THREAD

**Issue by shkleinik** [golang-migrate/migrate#338]
**Is your feature request related to a problem? Please describe.**
This repo currently depends on old and vulnerable `github.com/docker/docker` version: https://www.sourceclear.com/vulnerability-database/libraries/github-com-docker-docker/GO/GO/lid-886114/versions/v0.7.3
https://github.com/golang-migrate/migrate/blob/master/go.mod#L15

**Describe the solution you'd like**
Update to the non-vulnerable version (currently it is `1.13.1`) by running:
`go get -u github.com/docker/docker`

**Comment by dhui** [comment]
For those who are following, this is **not a critical security issue** since [dktest](https://github.com/dhui/dktest) is only used by `migrate` for tests.

The upstream library (dktest) [has been fixed](https://github.com/dhui/dktest/releases/tag/v0.3.2), and `migrate` will be updated accordingly sometime in the future.

**Comment by shkleinik** [comment]
@dhui Thanks a lot for quickly addressing this issue! I really appreciate it ^_^!

I understand, that in real life scenarios it is not a security concern.

However, synthetic security tests like Source Clear don't see the difference. Recently we migrated number (~10) of our `go` services from `golang/dep` to use `go` modules and now our builds are on fire. Under the hood Source Clear is using `go mod graph` which discovers _all the dependencies_ and unlike with `godep` dependency overriding doesn't help:
```
# go.mod
replace (
  github.com/docker/docker => github.com/docker/docker v1.3.3 
)
```
This replacement will make `go.sum` happy, but `go mod graph` still would spit out version before override.

**Comment by dhui** [comment]
FYI, master is now using an updated version of `dktest`

**Comment by dhui** [comment]
Fixed in [v4.9.0](https://github.com/golang-migrate/migrate/releases/tag/v4.9.0)
